speechbrain/lobes/features.py

```diff
@@ -441,3 +448,146 @@ def _check_input_shape(self, shape):
                 "Leaf expects 2d or 3d inputs. Got " + str(len(shape))
             )
         return in_channels
+
+
+def upalign_value(x, to: int) -> int:
+    """If `x` cannot evenly divide `to`, round it up to the next value that
+    can."""
+
+    assert x >= 0
+
+    if (x % to) == 0:
+        return x
+
+    return x + to - (x % to)
+
+
+@dataclass
```

Comment:
Do we want some tests for all this code maybe? I feel like the forward should be tested.